### PR TITLE
feat: Add tunneling example for Fastify

### PR DESCRIPTION
**Pull request by GeekCornerGH**
Also takes getsentry/sentry-javascript#9047 in account.

Express PR when this one gets merged (I'd rather do 1 thing at a time).

### CODE REVIEW

File: tunneling/fastify/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+const Fastify = require("fastify");
+
+const fastify = Fastify({ logger: true, bodyLimit: 100 * 1048576 }); //TODO: adjust bodyLimit to suit your needs (Default value is 100mb to match cloudflare free plan limits)
```
Comment (mydea) [inline tunneling/fastify/index.js:3]:
```suggestion
// Allow body up to 100 MB
const fastify = Fastify({ bodyLimit: 100 * 1048576 });
```

I think this is fine like this!

File: tunneling/fastify/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const SENTRY_KNOWN_PROJECTS = ["123456"]
+
+//TODO: Put your routes here, make sure to keep the /bugs route at the end of your file
+
+
+//Handle replay packets where no content-type header is available
+fastify.addContentTypeParser("*", { parseAs: "buffer" }, function (req, body, done) {
+    done(null, body);
+});
+fastify.post("/bugtracker", async (request, reply) => {
```
Comment (mydea) [inline tunneling/fastify/index.js:17]:
```suggestion
fastify.post("/bugs", async (request, reply) => {
```

to keep this consistent

File: tunneling/fastify/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        const piece = envelope.toString().split("\n")[0];
+        const header = JSON.parse(piece);
+
+        const dsn = new URL(header.dsn);
+        if (dsn.hostname !== SENTRY_HOST) {
+            return reply.code(400).send({ message: `Invalid Sentry host: ${dsn.hostname}` });
+        }
+
+        const project_id = dsn.pathname.substring(1);
+        console.log(project_id);
```
Comment (mydea) [inline tunneling/fastify/index.js:30]:
```suggestion
```

probably a leftover ;)

**Review by mydea — APPROVED** [review]
Looks good to me! Thanks a lot 🎉

File: tunneling/fastify/index.js (hunk 2 above)
Comment (GeekCornerGH) [reply tunneling/fastify/index.js:17]: I forgot sorry

**Comment by GeekCornerGH** [comment]
@mydea Can I get the `hacktoberfest-label`  as it got discussed in the issue? This is required in order for the pr to be taken in account on the hacktoberfest website

**Comment by GeekCornerGH** [comment]
Seems you don't need to do anything
image

**Pull request merged**